<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Special Effect Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="shadowTab">
      <attribute name="title">
       <string>Shadow</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="4" column="0">
        <widget class="QLabel" name="shadow_radius_label">
         <property name="text">
          <string>The blur radius for shadows</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="shadow_radius"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="shadow_offset_x_label">
         <property name="text">
          <string>The left offset for shadows</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="shadow_offset_x">
         <property name="minimum">
          <number>-99</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="shadow_offset_y_label">
         <property name="text">
          <string>The top offset for shadows</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="shadow_offset_y">
         <property name="minimum">
          <number>-99</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="shadow_opacity_label">
         <property name="text">
          <string>The translucency for shadows</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Color of shadow</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="shadow_color">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="clear_shadow">
         <property name="toolTip">
          <string>Fix some weirdness with ARGB windows</string>
         </property>
         <property name="text">
          <string>Zero the part of the shadow's mask behind the window</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="no_dnd_shadow">
         <property name="text">
          <string>Don't draw shadows on DND windows</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="no_dock_shadow">
         <property name="text">
          <string>Avoid drawing shadows on dock/panel windows</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="shadow">
         <property name="text">
          <string>Enabled client-side shadows on windows</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QCheckBox" name="shadow_ignore_shaped">
         <property name="text">
          <string>Avoid drawing shadow on all shaped windows</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="shadow_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="opacityTab">
      <attribute name="title">
       <string>Opacity</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="menu_opacity_label">
         <property name="text">
          <string>The opacity for menus</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="inactive_opacity_label">
         <property name="text">
          <string>Default opacity of inactive windows</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="active_opacity_label">
         <property name="text">
          <string>Default opacity for active windows</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="frame_opacity_label">
         <property name="text">
          <string>Opacity of window titlebars and borders</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="alpha_step_label">
         <property name="text">
          <string>XRender backend: Step size for alpha pictures
(Increasing it may result in less X resource usage)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="inactive_dim_label">
         <property name="text">
          <string>Dim inactive windows</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="inactive_dim_fixed">
         <property name="text">
          <string>Do not let dimness adjust based on window opacity</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="blur_background">
         <property name="text">
          <string>Blur background of transparent windows</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QCheckBox" name="blur_background_frame">
         <property name="text">
          <string>Blur background of opaque windows with transparent frames as well</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QCheckBox" name="blur_background_fixed">
         <property name="text">
          <string>Do not let blur radius adjust based on window opacity</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="active_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="inactive_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="menu_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="frame_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="alpha_step">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="inactive_dim">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="vsyncTab">
      <attribute name="title">
       <string>VSync</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="vsync_aggressive">
         <property name="toolTip">
          <string>Attempt to send painting request before VBlank and do XFlush()
  during VBlank. This switch may be lifted out at any moment.
</string>
         </property>
         <property name="text">
          <string>Aggressive</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="sw_opti">
         <property name="toolTip">
          <string>  Limit compton to repaint at most once every 1 / refresh_rate
  second to boost performance.
</string>
         </property>
         <property name="text">
          <string>Limit repainting</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="dbe">
         <property name="toolTip">
          <string>Enable DBE painting mode, intended to use with VSync to
  (hopefully) eliminate tearing.
</string>
         </property>
         <property name="text">
          <string>DBE</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="glx_no_stencil">
         <property name="toolTip">
          <string>GLX backend: Avoid using stencil buffer. Might cause issues
  when rendering transparent content. My tests show a 15% performance
  boost.</string>
         </property>
         <property name="text">
          <string>No GLX stencil</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="unredir_if_possible">
         <property name="toolTip">
          <string>  Unredirect all windows if a full-screen opaque window is
  detected, to maximize performance for full-screen windows.</string>
         </property>
         <property name="text">
          <string>Unredirect if possible</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="paint_on_overlay">
         <property name="toolTip">
          <string>  Painting on X Composite overlay window.
</string>
         </property>
         <property name="text">
          <string>Paint on overlay</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="vsync">
         <property name="toolTip">
          <string>  Set VSync method. There are (up to) 5 VSync methods currently
  available:
    none = No VSync
    drm = VSync with DRM_IOCTL_WAIT_VBLANK. May only work on some
      (DRI-based) drivers.
    opengl = Try to VSync with SGI_video_sync OpenGL extension. Only
      work on some drivers.
    opengl-oml = Try to VSync with OML_sync_control OpenGL extension.
      Only work on some drivers.
    opengl-swc = Try to VSync with SGI_swap_control OpenGL extension.
      Only work on some drivers. Works only with GLX backend.
    opengl-mswc = Try to VSync with MESA_swap_control OpenGL
      extension. Basically the same as opengl-swc above, except the
      extension we use.
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="otherTab">
      <attribute name="title">
       <string>Other</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Not implemented yet</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="aboutButton">
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>shadow</tabstop>
  <tabstop>no_dock_shadow</tabstop>
  <tabstop>no_dnd_shadow</tabstop>
  <tabstop>clear_shadow</tabstop>
  <tabstop>shadow_radius</tabstop>
  <tabstop>shadow_offset_x</tabstop>
  <tabstop>shadow_offset_y</tabstop>
  <tabstop>shadow_opacity</tabstop>
  <tabstop>shadow_color</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>menu_opacity</tabstop>
  <tabstop>inactive_opacity</tabstop>
  <tabstop>active_opacity</tabstop>
  <tabstop>frame_opacity</tabstop>
  <tabstop>alpha_step</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>inactive_dim</tabstop>
  <tabstop>inactive_dim_fixed</tabstop>
  <tabstop>blur_background</tabstop>
  <tabstop>blur_background_frame</tabstop>
  <tabstop>blur_background_fixed</tabstop>
  <tabstop>fading</tabstop>
  <tabstop>fade_delta</tabstop>
  <tabstop>no_fading_openclose</tabstop>
  <tabstop>fade_in_step</tabstop>
  <tabstop>fade_out_step</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MainDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MainDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
